test_fairlearn.py

```diff
@@ -1,118 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The reason why we kept the "redundant" test script was because it was giving more legible error messages than the new test scripts. New test scripts work nicely for automated checks, but didn't seem suitable for humans (at least not for me). My preference would be to keep this one around as a reference until we fix the legibility of new test scripts. (Maybe this was done, I'm just not aware.)